extensions/wikia/CommunityPage/CommunityPage.i18n.php

```diff
@@ -103,6 +106,9 @@
 	'communitypage-no-admins-contact' => 'Contact us to become the first admin text',
 	'communitypage-communitycentral-link' => 'Link to the community page for this language',
 	'communitypage-no-members' => 'Text for zero state of all members list',
+	'communitypage-popularpages-title' => 'Title for the Top Rising pages module',
+	'communitypage-popularpages-description' => 'Description for the Top Rising pages module',
+	'communitypage-lastrevision' => 'Details about user that last edited and time of editing',
```

Comment:
Messages with variables should have the variables explicitly documented in the QQQ strings so translators know exactly what `$1` and `$2` are.